tools/windows_portable/build_portable.ps1

```
@@ -539,54 +567,54 @@ function Install-PythonWheelArtifacts([string]$BundleRoot, [pscustomobject[]]$Ar
       throw "Python wheel artifact was not downloaded: $artifactId"
     }
     $sha256 = Get-RequiredStringProperty -Object $artifact -Name "sha256" -Context "artifact '$artifactId'"
+    $expectedBytes = [int64](Get-RequiredProperty -Object $artifact -Name "bytes" -Context "artifact '$artifactId'")
+    Assert-FileSize -FilePath $wheelPath -ExpectedBytes $expectedBytes
     Assert-Sha256 -FilePath $wheelPath -ExpectedHex $sha256
     uv pip install --reinstall --strict --no-deps --target $sitePackages $wheelPath
     Assert-LastExitCode -CommandLabel "uv pip install $artifactId"
   }
 }
 
 function Set-BundleRuntimeEnvironment([string]$BundleRoot) {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Set-BundleRuntimeEnvironment' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)